### PR TITLE
Add option to automatically create movie in plot_movie

### PR DESCRIPTION
**Pull request by micamu**
With pycbc_inference_plot_movie one can create frames to put them together into a movie. Because creating a movie requires having ffmpeg installed (which is not a pycbc requirement), we left that last step out and the user had to run it manually after generating the frames.
Here we add the option --make-movie, where the user can specify a path for creating the movie if ffmpeg is installed, and the option --cleanup, which will only work if --make-movie has been specified, with the purpose of removing the individual frames after creating the movie.

### CODE REVIEW

File: bin/inference/pycbc_inference_plot_movie [hunk 1]
```diff
@@ -116,6 +119,14 @@ parser.add_argument('--dpi', type=int, default=200,
 parser.add_argument("--nprocesses", type=int, default=None,
                     help="Number of processes to use. If not given then "
                          "use maximum.")
+parser.add_argument("--make-movie", type=str,
```
Comment (cmbiwer) [inline bin/inference/pycbc_inference_plot_movie]: This is a bad name for what this option means. Should be like ``--movie-file`` or something.

Same with ``--output-file``. Its not an output file. Should be like ``--output-dir``.

Comment (micamu) [inline bin/inference/pycbc_inference_plot_movie]: Sorry, didn't see that before! Will change it accordingly today

**Comment by cmbiwer** [comment]
@micamu Thanks, ``--movie-file`` is a much better name.

Its only my opinion but perhaps ``--output-file-prefix`` is more indicative for the other option. But if you don't want to change it, then its fine.

**Comment by micamu** [comment]
That sounds good, I'll use ```--output-prefix``` (slightly shorter)

**Pull request merged**